python/dask_cudf/dask_cudf/backends.py

```diff
@@ -473,6 +475,26 @@ def sizeof_cudf_series_index(obj):
     return obj.memory_usage()
 
 
+@normalize_token.register(cudf.core.frame.Frame)
+def _normalize_cudf_frame(obj):
+    # Only move data to pandas if we "need" to
+    if config.get("tokenize.ensure-deterministic", False):
+        return [
+            type(obj),
+            obj._dtypes,
+            (
+                obj.hash_values() if hasattr(obj, "hash_values") else obj
+            ).to_pandas(),
+        ]
+    # If deterministic tokens are not needed, return a random UUID4 value.
+    return uuid.uuid4().hex
+
+
+@normalize_token.register(cudf.core.index.RangeIndex)
+def _normalize_cudf_rangeindex(index):
+    return index.to_pandas()
```

Comment:
I guess this is cheap enough that it doesn't matter, but `(type(index), index.start, index.stop, index.step)` or similar would work too, right?

Reply:
Yeah, that works